manifests/service.pp

```diff
@@ -1,14 +1,14 @@
 # Class: iscsi::service
 #
 # This class manages the service for the iscsi module
-class iscsi::service () inherits iscsi::params {
+class iscsi::service () inherits iscsi::initiator {
```

Comment:
this inherits is actually obsolete, the class iscsi::service is a private class called from iscsi::initiatior so the variables I wnat to use should be known...